DigitalOffice/Services/AuthenticationService/Validators/UserLoginValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.AuthenticationService.Models;
+
+namespace LT.DigitalOffice.AuthenticationService.Validators
+{
+    public class UserLoginValidator : AbstractValidator<UserLoginCredentialsRequest>
+    {
+        public UserLoginValidator()
+        {
+            RuleFor(user => user.Email)
+                .NotEmpty();
```

Comment:
Maybe you should add a simple check EmailAdress() and add a test for it?

Reply:
it makes sense if all this is checked when creating a user?

Reply:
At least we will optimize the work with the database, because incorrect emails will not go through

Reply:
Then rewrite the entire validator from user creation here, I have already done this, but the guys above in the comments wrote that this should not be done